scripts/expected_api_mismatch.txt

```diff
@@ -12,3 +12,5 @@ Parameter type mismatch in BrowserContext.route(handler=): documented as Callabl
 Parameter type mismatch in BrowserContext.unroute(handler=): documented as Union[Callable[[Route, Request], Union[Any, Any]], None], code has Union[Callable[[Route, Request], Any], Callable[[Route], Any], None]
 Parameter type mismatch in Page.route(handler=): documented as Callable[[Route, Request], Union[Any, Any]], code has Union[Callable[[Route, Request], Any], Callable[[Route], Any]]
 Parameter type mismatch in Page.unroute(handler=): documented as Union[Callable[[Route, Request], Union[Any, Any]], None], code has Union[Callable[[Route, Request], Any], Callable[[Route], Any], None]
+
+Parameter type mismatch in Page.add_locator_handler(handler=): documented as Callable[[Locator], Any], code has Callable
```

Comment:
I think if we change that, its considered a breaking change (pyright), not sure if we can do it without breaking:

```text
  /Users/maxschmitt/Developer/playwright-python/tests/sync/test_page_add_locator_handler.py:189:66 - error: Argument of type "() -> None" cannot be assigned to parameter "handler" of type "(Locator) -> Any" in function "add_locator_handler"
    Type "() -> None" cannot be assigned to type "(Locator) -> Any"
      Function accepts too many positional parameters; expected 0 but received 1 (reportGeneralTypeIssues)
```